### PR TITLE
Move some utils from client to types

### PR DESCRIPTION
**Pull request by thibault-martinez**

### CODE REVIEW

**Review by Thoralf-M — DISMISSED** [review]

**Review by Thoralf-M — DISMISSED** [review]

**Review by Thoralf-M — APPROVED** [review]

File: types/src/block/output/alias_id.rs [hunk 1]
```diff
@@ -19,6 +22,11 @@ impl AliasId {
     pub fn or_from_output_id(self, output_id: &OutputId) -> Self {
         if self.is_null() { Self::from(output_id) } else { self }
     }
+
+    /// Returns the bech32 encoding of the alias ID.
+    pub fn to_bech32(&self, bech32_hrp: &str) -> String {
```
Comment (kwek20) [inline types/src/block/output/alias_id.rs:27]: maybe we should make this a trait?

Comment (thibault-martinez) [reply types/src/block/output/alias_id.rs:27]: Yeah I've been thinking about that as well, I don't like that Bech32 is not strongly typed. I'll open an issue. A topic for another day.

Comment (thibault-martinez) [reply types/src/block/output/alias_id.rs:27]: https://github.com/iotaledger/iota-sdk/issues/18

**Review by kwek20 — APPROVED** [review]
1 comment but fine to merge

**Pull request merged**